src/modules/comments.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable consistent-return */
+/* eslint-disable camelcase */
+class Comment {
+  constructor(item_id, username, comment) {
+    this.item_id = item_id;
+    this.username = username;
+    this.comment = comment;
+  }
+}
+
+const postComment = async (comment) => {
+  const posts = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/pRSMj4Sa1SZFpBHfoj5I/comments', {
+    method: 'POST',
```

Comment:
The functionality of adding a comment is not very consistent. Sometimes it works and sometimes it doesn't.

- The app actually behaves in an unusual manner, when I start by liking any tv show, then attempt to add a comment it works. But if I were to refresh the page, then try to add comments again, it doesn't work anymore.
- Kindly make sure that a comment can be added to all the TV Shows shows at all times. In general, adding comments works, but for some reason sometimes it decides not to work.